src/containers/PageBuilder/Primitives/SearchCTA/SearchCTA.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Form as FinalForm } from 'react-final-form';
+import { useHistory } from 'react-router-dom';
+import { createResourceLocatorString } from '../../../../util/routes';
+import { useRouteConfiguration } from '../../../../context/routeConfigurationContext';
+import { useConfiguration } from '../../../../context/configurationContext';
+import { FormattedMessage } from '../../../../util/reactIntl';
```

Comment:
```suggestion
// Contexts
import { useRouteConfiguration } from '../../../../context/routeConfigurationContext';
import { useConfiguration } from '../../../../context/configurationContext';
// Utils
import { FormattedMessage } from '../../../../util/reactIntl';
import { createResourceLocatorString } from '../../../../util/routes';
```

Perhaps it would be a bit cleaner and futureproof to keep the order of different imports constant. Like this:
https://github.com/sharetribe/web-template/blob/main/src/containers/ListingPage/ListingPageCarousel.js#L6-L12